configuration/configuration.go

```diff
@@ -311,6 +311,13 @@ type DataConfiguration struct {
 	// useful to just try to fetch all blocks before checking for balance
 	// consistency.
 	BalanceTrackingDisabled bool `json:"balance_tracking_disabled"`
+
+	// EndAtTip is an end condition. syncing will stop once tip is reached
```

Comment:
Can we nest these in an `EndConditions` object? I think we will have many end conditions?